README.md

```diff
@@ -1541,6 +1541,7 @@ Update Time, five active automations, webhooks.
   * [landen.co](https://www.landen.co) — Generate, edit, and publish beautiful websites and landing pages for your startup. All without code. The free tier allows you to have one website, fully customizable and published on the web.
   * [Quant Ux](https://quant-ux.com/) - Quant Ux is a prototyping and design tool. - It's completely free and also open source.
   * [lensdump.com](https://lensdump.com/) - Free cloud image hosting.
+  * [Logoman](https://logoman.pages.dev/) - Free, no-frills logo maker for solofounders.
```

Comment:
this appears to just put a free font on a background, that's not really a service sorry